app/graphql/types/system.rb

```diff
@@ -3,12 +3,13 @@
 module Types
   # Definition of the System GraphQL type
   class System < Types::BaseObject
+    model_class ::Host
```

Comment:
Can we not rename `System` to `Host` or the other way around?

Reply:
We can definitely do that in some other PR - the reasoning behind this is that a while ago, there were meant to be two classes of systems, `Host` and `Imagestream` - but that's been delayed for a while